.azure-pipelines/build-template.yml

```diff
@@ -75,26 +81,6 @@ jobs:
           libteam5 \
           libteamdctl0
     displayName: "Install dependencies"
-  - task: DownloadPipelineArtifact@2
```

Comment:
As discussed, let's bring this back as we would need to download from respective repo.